modules/shortcodes.php

```diff
@@ -134,4 +134,13 @@ function jetpack_shortcode_get_wpvideo_id( $atts ) {
 	}
 }
 
+if ( ! function_exists( 'jetpack_shortcode_get_videopress_id' ) ) {
+	function jetpack_shortcode_get_videopress_id( $atts ) {
+		if ( isset( $atts[ 0 ] ) )
```

Comment:
Done in 8c70190f2e4d03eee44c1773651da36fe0e3c4bd, along with quick changes for coding standards in the whole file.